<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface</class>
 <widget class="QMainWindow" name="Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1146</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interface</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="show_res">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>951</width>
      <height>241</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
 font-size:20px;
font-weight:bold;
color:rgb(255, 255, 255);
letter-spacing:5px;
outline-color:rgb(255, 255, 255);
border-radius:10px;
background-color:rgba(0, 170, 127, 100)

}

background-color : white;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>951</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_mod">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

font-size:18px;
font-weight:bold;
color:rgb(255, 255, 255);
letter-spacing:5px;
outline-color:rgb(255, 255, 255);
border-radius:10px;
background-color:rgba(0, 170, 127, 100)
}</string>
       </property>
       <property name="text">
        <string>mod</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_ln">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

font-size:18px;
font-weight:bold;
color:rgb(255, 255, 255);
letter-spacing:5px;
outline-color:rgb(255, 255, 255);
border-radius:10px;
background-color:rgba(0, 170, 127, 100)
}</string>
       </property>
       <property name="text">
        <string>ln</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton_log">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

font-size:18px;
font-weight:bold;
color:rgb(255, 255, 255);
letter-spacing:5px;
outline-color:rgb(255, 255, 255);
border-radius:10px;
background-color:rgba(0, 170, 127, 100)
}</string>
       </property>
       <property name="text">
        <string>log</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="pushButton_exp">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

font-size:20px;
font-weight:bold;
color:rgb(255, 255, 255);
letter-spacing:5px;
outline-color:rgb(255, 255, 255);
border-radius:10px;
background-color:rgba(0, 170, 127, 100)
}</string>
       </property>
       <property name="text">
        <string>^</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_sin">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

font-size:18px;
font-weight:bold;
color:rgb(255, 255, 255);
letter-spacing:5px;
outline-color:rgb(255, 255, 255);
border-radius:10px;
background-color:rgba(0, 170, 127, 100)
}</string>
       </property>
       <property name="text">
        <string>sin</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_cos">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

font-size:18px;
font-weight:bold;
color:rgb(255, 255, 255);
letter-spacing:5px;
outline-color:rgb(255, 255, 255);
border-radius:10px;
background-color:rgba(0, 170, 127, 100)
}</string>
       </property>
       <property name="text">
        <string>cos</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_tan">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

font-size:18px;
font-weight:bold;
color:rgb(255, 255, 255);
letter-spacing:5px;
outline-color:rgb(255, 255, 255);
border-radius:10px;
background-color:rgba(0, 170, 127, 100)
}</string>
       </property>
       <property name="text">
        <string>tan</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_clean">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color:red;;
font-size:20px;
color:rgb(255, 255, 255);
font-weight:bold;
border-radius: 10px;
}
</string>
       </property>
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="pushButton_acos">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

font-size:16px;
font-weight:bold;
color:rgb(255, 255, 255);
letter-spacing:5px;
outline-color:rgb(255, 255, 255);
border-radius:10px;
background-color:rgba(0, 170, 127, 100)
}</string>
       </property>
       <property name="text">
        <string>acos</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_7">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color:rgb(0, 170, 127);
font-size:20px;
color:rgb(255, 255, 255);
font-weight:bold;
border-radius: 10px;

}</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_8">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color:rgb(0, 170, 127);
font-size:20px;
color:rgb(255, 255, 255);
font-weight:bold;
border-radius: 10px;

}</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_9">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color:rgb(0, 170, 127);
font-size:20px;
color:rgb(255, 255, 255);
font-weight:bold;
border-radius: 10px;

}</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_dividion">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

font-size:20px;
font-weight:bold;
color:rgb(255, 255, 255);
letter-spacing:5px;
outline-color:rgb(255, 255, 255);
border-radius:10px;
background-color:rgba(0, 170, 127, 100)
}</string>
       </property>
       <property name="text">
        <string>/</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="pushButton_asin">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

font-size:16px;
font-weight:bold;
color:rgb(255, 255, 255);
letter-spacing:5px;
outline-color:rgb(255, 255, 255);
border-radius:10px;
background-color:rgba(0, 170, 127, 100)
}</string>
       </property>
       <property name="text">
        <string>asin</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_4">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color:rgb(0, 170, 127);
font-size:20px;
color:rgb(255, 255, 255);
font-weight:bold;
border-radius: 10px;

}</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButton_5">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color:rgb(0, 170, 127);
font-size:20px;
color:rgb(255, 255, 255);
font-weight:bold;
border-radius: 10px;

}</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_6">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color:rgb(0, 170, 127);
font-size:20px;
color:rgb(255, 255, 255);
font-weight:bold;
border-radius: 10px;

}</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton_minus">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

font-size:20px;
font-weight:bold;
color:rgb(255, 255, 255);
letter-spacing:5px;
outline-color:rgb(255, 255, 255);
border-radius:10px;
background-color:rgba(0, 170, 127, 100)
}</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="pushButton_atan">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

font-size:16px;
font-weight:bold;
color:rgb(255, 255, 255);
letter-spacing:5px;
outline-color:rgb(255, 255, 255);
border-radius:10px;
background-color:rgba(0, 170, 127, 100)
}</string>
       </property>
       <property name="text">
        <string>atan</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_1">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color:rgb(0, 170, 127);
font-size:20px;
color:rgb(255, 255, 255);
font-weight:bold;
border-radius: 10px;

}</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color:rgb(0, 170, 127);
font-size:20px;
color:rgb(255, 255, 255);
font-weight:bold;
border-radius: 10px;

}</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="pushButton_3">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color:rgb(0, 170, 127);
font-size:20px;
color:rgb(255, 255, 255);
font-weight:bold;
border-radius: 10px;

}</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="pushButton_plus">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

font-size:20px;
font-weight:bold;
color:rgb(255, 255, 255);
letter-spacing:5px;
outline-color:rgb(255, 255, 255);
border-radius:10px;
background-color:rgba(0, 170, 127, 100)
}</string>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="4" column="4">
      <widget class="QPushButton" name="pushButton_graff">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

font-size:15px;
font-weight:bold;
color:rgb(25, 0, 255);
letter-spacing:5px;
outline-color:rgb(255, 255, 255);
border-radius:10px;
background-color:rgba(200, 70, 127, 100)
}</string>
       </property>
       <property name="text">
        <string>Graff</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="pushButton_0">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color:rgb(0, 170, 127);
font-size:20px;
color:rgb(255, 255, 255);
font-weight:bold;
border-radius: 10px;

}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="pushButton_e">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background-color:rgb(0, 170, 127);
font-size:20px;
color:rgb(255, 255, 255);
font-weight:bold;
border-radius: 10px;

}</string>
       </property>
       <property name="text">
        <string>e</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="pushButton_dot">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

background-color:rgb(0, 170, 127);
font-size:20px;
color:rgb(255, 255, 255);
font-weight:bold;
border-radius: 10px;
}
</string>
       </property>
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="5" column="3">
      <widget class="QPushButton" name="pushButton_multi">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

font-size:20px;
font-weight:bold;
color:rgb(255, 255, 255);
letter-spacing:5px;
outline-color:rgb(255, 255, 255);
border-radius:10px;
background-color:rgba(0, 170, 127, 100)
}</string>
       </property>
       <property name="text">
        <string>*</string>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <widget class="QPushButton" name="pushButton_credit">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

font-size:15px;
font-weight:bold;
color:rgb(255, 140, 130);
letter-spacing:5px;
outline-color:rgb(255, 255, 255);
border-radius:10px;
background-color:rgba(0, 170, 7, 100)
}</string>
       </property>
       <property name="text">
        <string>Gredit</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="pushButton_x">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

background-color:rgba(200, 70, 127, 100);
font-size:20px;
color:rgb(0, 0, 255);
font-weight:bold;
border-radius: 10px;
}
</string>
       </property>
       <property name="text">
        <string>x</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QPushButton" name="pushButton_left_br">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

background-color:rgb(255, 170, 127);
font-size:20px;
color:rgb(255, 255, 255);
font-weight:bold;
border-radius: 10px;
}
</string>
       </property>
       <property name="text">
        <string>(</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="pushButton_rigth_br">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

background-color:rgb(255, 170, 127);
font-size:20px;
color:rgb(255, 255, 255);
font-weight:bold;
border-radius: 10px;
}
</string>
       </property>
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item row="6" column="3">
      <widget class="QPushButton" name="pushButton_equal">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
font-size:28px;
font-weight:bold;
color:rgb(255, 255, 255);
letter-spacing:5px;
outline-color:rgb(255, 255, 255);
border-radius:10px;
background-color:red;

}</string>
       </property>
       <property name="text">
        <string>=</string>
       </property>
      </widget>
     </item>
     <item row="6" column="4">
      <widget class="QPushButton" name="pushButton_sqrt">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

font-size:16px;
font-weight:bold;
color:rgb(255, 255, 255);
letter-spacing:5px;
outline-color:rgb(255, 255, 255);
border-radius:10px;
background-color:rgba(0, 170, 127, 100)
}</string>
       </property>
       <property name="text">
        <string>sqrt</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_clear">
       <property name="maximumSize">
        <size>
         <width>1677719</width>
         <height>1677721</height>
        </size>
       </property>
       <property name="tabletTracking">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {

font-size:28px;
font-weight:bold;
color:red;
letter-spacing:5px;
outline-color:rgb(255, 255, 255);
border-radius:10px;
background-color:rgba(0, 170, 127, 100)
}</string>
       </property>
       <property name="text">
        <string>AC</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_dividion_set_x">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>670</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>5500</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {

background-color:rgba(200, 70, 127, 100);
font-size:20px;
color:rgb(0, 0, 255);
font-weight:bold;
border-radius: 10px;
}</string>
    </property>
    <property name="text">
     <string>set x</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1146</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
